validation/src/main/scala/hmda/validation/rules/lar/validity/V385.scala

```diff
@@ -0,0 +1,31 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+
+object V385 extends EditCheck[LoanApplicationRegister] {
+
+  val requiredDenialReasons = (1 to 9).map(_.toString()).toList
+  val validDenialReasons = List("") ::: requiredDenialReasons
+  val okActionTaken = List(3, 7)
+
+  def apply(lar: LoanApplicationRegister): Result = {
+    val denialReasons = List(lar.denial.reason1, lar.denial.reason2, lar.denial.reason3)
+
+    /*
+        denialReasons.foreach(_ is containedIn(validDenialReasons)) and
+     */
+
+    (denialReasons(0) is containedIn(validDenialReasons)) and
```

Comment:
Since this edit requires you to make two totally unrelated checks (first that the input is 1-9 or blank, second that there is a denial reason when necessary), I think it would be more readable if you extracted each of those into a named method. You can see the strategy I'm thinking of [here](https://github.com/cfpb/hmda-platform/pull/296/files#diff-d6104f317036ac47686bff04753c8795R34).